### PR TITLE
chore(auditlogging): add a test to reproduce duplicate resource in logging

### PR DESCRIPTION
**Pull request by moogacs**
### Review checklist

- [ ] Documentation has been updated, if necessary. Link to changed documentation:
- [ ] Chaos pipeline run or not necessary. Link to pipeline:
- [ ] All new code is covered by tests where it is reasonable.
- [ ] Performance tests have been run or not necessary.

### CODE REVIEW

File: usecases/auth/authorization/rbac/authorizer_test.go [hunk 1]
```diff
@@ -470,6 +469,149 @@ func TestFilterAuthorizedResourcesLogging(t *testing.T) {
 		"Allowed resources should match input resources")
 }
 
+func TestAuthorizeResourceAggregation(t *testing.T) {
+	// Setup proper logger with hook for testing
+	logger := logrus.New()
```
Comment (antas-marcin) [inline usecases/auth/authorization/rbac/authorizer_test.go:474]: you could use `logrustest "github.com/sirupsen/logrus/hooks/test"`

then the hook is created by default:
```
logger, loggerHook := logrustest.NewNullLogger()
logger.SetLevel(logrus.InfoLevel)
```

File: usecases/auth/authorization/rbac/authorizer_test.go [hunk 2]
```diff
@@ -470,6 +469,149 @@ func TestFilterAuthorizedResourcesLogging(t *testing.T) {
...
+	assert.Len(t, resources, 1, "should return 1 unique resource (duplicates are aggregated)")
+
+	results, ok := perm["results"].(string)
+	require.True(t, ok, "results should be a string")
+	assert.Equal(t, "success", results)
+}
+
+func TestFilterAuthorizedResourcesAggregation(t *testing.T) {
+	// Setup proper logger with hook for testing
+	logger := logrus.New()
```
Comment (antas-marcin) [inline usecases/auth/authorization/rbac/authorizer_test.go:544]: same here, I would switch to `"github.com/sirupsen/logrus/hooks/test"`

File: usecases/auth/authorization/rbac/authorizer_test.go (hunk 1 above)
Comment (moogacs) [reply usecases/auth/authorization/rbac/authorizer_test.go:474]: i actually made it in purpose to see the logs in the test

Comment (moogacs) [reply usecases/auth/authorization/rbac/authorizer_test.go:474]: we discussed internally and agreed to keep it
